pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -797,12 +926,12 @@ template <class TensorNetT> void lightningTensorClassBindings(py::module_ &m) {
     /* Observables submodule */
     py::module_ obs_submodule =
         m.def_submodule("observables", "Submodule for observables classes.");
-    registerBackendAgnosticObservables<TensorNetT>(obs_submodule);
+    registerBackendAgnosticObservablesTensor<TensorNetT>(obs_submodule, name);
 
     //***********************************************************************//
     //                             Measurements
     //***********************************************************************//
-    class_name = "MeasurementsC" + bitsize;
+    class_name = std::string(name) + "MeasurementsC" + bitsize;
```

Comment:
Do we have to update the class name? Could we just instantiate  `MeasurementsTNCuda<TensorNetT>` with different `exacttn` and `mps` class?

Reply:
I tried and compiled without problem but when I call the module show this error:
``` bash
ImportError: generic_type: cannot initialize type "NamedObsC64": an object with that name is already defined
```
that is why we need to set unique names for each module